.prettierrc.json

```diff
@@ -0,0 +1,20 @@
+{
+  "arrowParens": "always",
+  "tabWidth": 2,
+  "singleQuote": true,
+  "printWidth": 100,
+  "semi": false,
+  "trailingComma": "es5",
+  "useTabs": false,
+  "plugins": [
+    "@trivago/prettier-plugin-sort-imports"
+  ],
+  "importOrder": [
+    "^(^react$|@react|react)",
+    "<THIRD_PARTY_MODULES>",
+    "^@/(.*)$",
+    "^[./]"
+  ],
+  "importOrderSeparation": true,
+  "importOrderSortSpecifiers": true
```

Comment:
These are configs from the [@trivago/prettier-plugin-sort-imports](https://github.com/trivago/prettier-plugin-sort-imports) plugin.

It sorts imports by:
1. from React
2. Third-party modules
3. Imports from aliases (you could customize the regexp for your liking, e.g: @/components, @hooks, etc)
4. Regular imports